app/views/reports/regions/show.html.erb

```diff
@@ -244,7 +244,7 @@
                row_resource_display_field: :full_name,
                row_resource_link: :admin_user_path %>
   <% else %>
-    <%= render "analytics/facilities/analytics_table",
+    <%= render "new_registrations_table",
```

Comment:
I think we can call this `registrations_table` for consistency, what do you think?

Reply:
There already is a `registrations_table` in mix here -- we are running out of names. 😁